### PR TITLE
translations and minor layout fixes

### PR DESCRIPTION
**Pull request by carsten19**
- association selects are missing translations
- network map tries to load the js.map (can be seen as 404 in the browser)
- reduce size of network map a bit so that it fits
- sort devices by node ID so that the sequence is correct. If you currently refresh a device it is removed and added again, jumping to the end of the list although it might have a lower node ID
- disable the source node of an association in the target node select list (associations in the same node don't make sense)

### CODE REVIEW

File: admin/i18n/de/translations.json [hunk 1]
```diff
@@ -22,6 +23,7 @@
     "Product": "Produkt",
     "Remove node": "Gerät entfernen",
     "Save Config:": "Konfiguration speichern:",
+    "Select group": "Gruppe wählen",
```
Comment (AlCalzone) [inline admin/i18n/de/translations.json]: Um konsistent zu bleiben:
```suggestion
    "Select group": "Gruppe auswählen",
```

words.js nicht vergessen!

File: admin/index.html [hunk 2]
```diff
@@ -387,6 +387,9 @@
         });
         
         socket.emit('getObjectView', 'system', 'device', {startkey: 'zwave.' + instance + '.', endkey: 'zwave.' + instance + '.\u9999', include_docs: true}, function (err, _devices) {
+            _devices.rows.sort((a, b) => {
+                return (a.value.native.nodeID < b.value.native.nodeID ? -1 : 1);
+            });
```
Comment (AlCalzone) [inline admin/index.html]: Ich mag Arrow-Functions mit return nicht, wenns auch als Einzeiler geht.
```js
_devices.rows.sort((a, b) => (a.value.native.nodeID < b.value.native.nodeID ? -1 : 1));
```

File: admin/index.html [hunk 3]
```diff
@@ -428,6 +430,7 @@
                                 }
 
                                 if (obj.result && obj.result !== "no groups") {
+                                    $('#assGroups').append($("<option>").attr("disabled", "disabled").text(_('Select group'))).selectmenu('refresh');
```
Comment (AlCalzone) [inline admin/index.html:433]: Das braucht einen Kommentar

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: admin/i18n/de/translations.json (hunk 1 above)
Comment (carsten19) [reply admin/i18n/de/translations.json]: done

File: admin/index.html (hunk 2 above)
Comment (carsten19) [reply admin/index.html]: done

File: admin/index.html (hunk 3 above)
Comment (carsten19) [reply admin/index.html:433]: done

**Review by AlCalzone — APPROVED** [review]

**Comment by carsten19** [comment]
@AlCalzone when you merge that could you also bump up the version?

**Comment by carsten19** [comment]
When will that PR be merged?

**Comment by GermanBluefox** [comment]
@AlCalzone LGTM ?

**Pull request merged**